src/p_saveg.c

```diff
@@ -941,11 +941,19 @@ static void saveg_read_player_t(player_t *str)
 
     if (saveg_compat > saveg_woof600)
     {
+        // [Woof!]: fixed_t pitch;
+        str->pitch = saveg_read32();
```

Comment:
Should `oldpitch` get set to the same value?

Reply:
I think it is reset in `P_PlayerThink`. We can save and restore `oldpitch` as we do with other old* fields.